pkg/controller/windowsmachineconfig/windowsmachineconfig_controller.go

```diff
@@ -255,21 +259,25 @@ func (r *ReconcileWindowsMachineConfig) createWindowsWorkerNodes(count int) bool
 	var errs []error
 	for i := 0; i < count; i++ {
 		// Create Windows VM in the cloud provider
+		log.V(1).Info("creating the Windows VM")
```

Comment:
why are we using log instead of r.log?

Reply:
Sorry I missed that. I also realize I have missed fixing the `log.Error()` calls. I will fix those too to use the global variable that I will introduce.